config/crd/bases/anywhere.eks.amazonaws.com_tinkerbellmachineconfigs.yaml

```diff
@@ -42,7 +42,7 @@ spec:
                 additionalProperties:
                   type: string
                 description: HardwareSelector models a simple key-value selector used
-                  in Tinkerbell providioning.
+                  in Tinkerbell provisioning.
```

Comment:
Can we run `make release-manifests`? 
that would update the eksa-components.yaml